test/python/T0_t/Config/Tier0Config_t.py

```diff
@@ -0,0 +1,557 @@
+import unittest
+import T0.RunConfig.Tier0Config as tier0config
+
+
+class TestRetrieveStreamConfig(unittest.TestCase):
+
+    def setUp(self):
+        self.config = tier0config.createTier0Config()
+
+    def test_createTier0Config(self):
+        # Check some key values in the Global section
+        self.assertEqual(self.config.Global.ProcessingSite, "T2_CH_CERN")
+        self.assertEqual(self.config.Global.StreamerPNN, "T0_CH_CERN_Disk")
+        self.assertEqual(self.config.Global.StorageSite, "T0_CH_CERN_Disk")
+        self.assertEqual(self.config.Global.DQMDataTier, "DQMIO")
+        self.assertEqual(self.config.Global.BaseRequestPriority, 150000)
+        self.assertEqual(self.config.Global.EnableUniqueWorkflowName, False)
+        self.assertEqual(self.config.Global.DeploymentID, 1)
+
+    def test_retrieveStreamConfig(self):
+        # Test with a non-existing stream, it should fall back to "Default"
+        nonExistentStreamName = "NonExistentStream"
+        streamConfig = tier0config.retrieveStreamConfig(self.config, nonExistentStreamName)
+
+        # Check if the default values were copied correctly
+        self.assertEqual(streamConfig._internal_name, nonExistentStreamName)
+        self.assertEqual(getattr(self.config.Streams, nonExistentStreamName), streamConfig)
+
+    def test_deleteStreamConfig(self):
+        # Test with an existing stream
+        streamName = "Express"
+        tier0config.deleteStreamConfig(self.config, streamName)
```

Comment:
AS there was no `Express` stream in the first place, this test would succeed even if the function does not work. We could thest this after testing addRepackConfig, so there is a stream to delete.